pkg/cmd/pipeline/start.go

```diff
@@ -159,6 +160,7 @@ like cat,foo,bar
 	c.Flags().StringSliceVarP(&opt.Labels, "labels", "l", []string{}, "pass labels as label=value.")
 	c.Flags().BoolVarP(&opt.DryRun, "dry-run", "", false, "preview pipelinerun without running it")
 	c.Flags().StringVarP(&opt.Output, "output", "", "", "format of pipelinerun dry-run (yaml or json)")
+	c.Flags().StringVarP(&opt.Name, "name", "", "", "specify a name for the pipelinerun (must be lowercase alphanumeric characters)")
```

Comment:
Should it be call `prefixName` or something ? just `name` would either make me think it's the `pipeline` name or will be the full `pipelinerun` name :sweat_smile:

Reply:
Yeah, I was wondering about this myself so thanks for mentioning. Think `--prefix-name` works.

Reply:
sounds good to me too!